### PR TITLE
Slower Potted Plants

### PR DESCRIPTION
**Pull request by Robustin**
:cl: Robustin
tweak: Carrying a potted plant will now slow you down
/:cl:

I honestly thought the plants were a one-off meme that would be removed but since it looks like they're here to stay, how about we make them a little less ridiculous.

### CODE REVIEW

File: code/game/objects/structures/flora.dm [hunk 1]
```diff
@@ -234,6 +234,7 @@
 	throwforce = 13
 	throw_speed = 2
 	throw_range = 4
+	slowdown = 2
```
Comment (ChangelingRain) [inline code/game/objects/structures/flora.dm]: Needs `flags = HANDSLOW` too

**Comment by KorPhaeron** [comment]
already rejected previously sorry

**Comment by Robustin** [comment]
@KorPhaeron 

Where was this rejected, I couldn't find anything.

**Comment by KorPhaeron** [comment]
https://github.com/tgstation/tgstation/pull/19546

**Comment by PKPenguin321** [comment]
the delicate potted plant meta

**Pull request closed without merging**